src/pages/tutorial/cw-contract/query.mdx

```diff
@@ -0,0 +1,323 @@
+# Creating a query
+
+We have already created a simple contract reacting to an empty instantiate message. Unfortunately,
+it is not very useful. Let's make it a bit reactive.
```

Comment:
I don't know if "reactive" is a good term here

Reply:
Changed to `interactive`